projects/eval-adis/src/examples/examples_src.mk

```diff
@@ -7,6 +7,7 @@ endif
 ifeq (y,$(strip $(IIO_TRIGGER_EXAMPLE)))
 TINYIIOD=y
 CFLAGS += -DIIO_TRIGGER_EXAMPLE=1
+CFLAGS += -O2
```

Comment:
I think it would be better to add a `RELEASE` make flag, so this flag will be set in generic.mk.